### PR TITLE
Add stinj

### PR DESCRIPTION
**Pull request by riley-stride**

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -2570,6 +2570,13 @@ const chainInfos = (
           coinGeckoId: "pool:staevmos",
           coinImageUrl: "/tokens/stevmos.svg",
         },
+        {
+          coinDenom: "stINJ",
+          coinMinimalDenom: "stinj",
+          coinDecimals: 18,
+          coinGeckoId: "",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:2577]: the pool creator may use the preview generated in this PR to create the pool, but can you please specify an 'id' and define its price in 'config/prices.ts'?

File: packages/web/config/ibc-assets.ts [hunk 2]
```diff
@@ -744,6 +744,12 @@ export const IBCAssetInfos: (IBCAsset & {
         destChannelId: "channel-5",
         coinMinimalDenom: "staevmos",
       },
+      {
+        counterpartyChainId: "stride-1",
+        sourceChannelId: "channel-326",
+        destChannelId: "channel-5",
+        coinMinimalDenom: "stinj",
+      },
```
[the remark this answers is not in the document]
Comment (riley-stride) [reply packages/web/config/ibc-assets.ts:752]: oh, very cool! done

File: packages/web/config/chain-infos.ts (hunk 1 above)
Comment (riley-stride) [reply packages/web/config/chain-infos.ts:2577]: I don't think there's an stINJ pool yet, should I leave the `poolId` field in `price.ts` empty?

Comment (JeremyParish69) [reply packages/web/config/chain-infos.ts:2577]: Sure, we'll just set this PR to draft until there is a pool we can put here.

Comment (JeremyParish69) [reply packages/web/config/chain-infos.ts:2577]: Hey @riley-stride just want to bump on this. Any update on initial liquidity so we can list the token?

**Comment by jonator** [comment]
Closing as stale, can be reopened later if needed

**Pull request closed without merging**